tasks/install_shell.sh

```diff
@@ -432,6 +437,19 @@ do_fetch() {
   return 0
 }
 
+do_apt_helper() {
+  info "Trying apt-helper..."
+  run_cmd "/usr/lib/apt/apt-helper download-file \"$1\" \"$2\"" 2>$tmp_stderr
```

Comment:
Do we need double quotes? 
```suggestion
  run_cmd "/usr/lib/apt/apt-helper download-file '$1' '$2'" 2>$tmp_stderr
```

Reply:
updated